### PR TITLE
Adding tutorial and doctree ref

### PR DESCRIPTION
**Pull request by JonasAls89**
Skjermdump fra 2022-05-06 14-09-11

### CODE REVIEW

File: tutorial-getting-started-collect.rst [hunk 1]
```diff
@@ -5,28 +5,979 @@
 Collect data
 ============
 
-In this tutorial we will look closer into how to...
+In this tutorial we will look closer at how to connect to the systems HubSpot and Twelvedata. `HubSpot <https://www.hubspot.com/>`_ is a CRM system whilst `Twelvedata <https://twelvedata.com/>`_ is a system containing financial data such as stock information. After having succesfully connected to these systems you will create inbound pipes for each relevant datatype we want to work on as we go through a Sesam dataflow in the getting started guide.
```
Comment (erik-leven) [inline tutorial-getting-started-collect.rst:8]: Hmm, again HubSpot and Twelvedata are not systems in this case. I think "data provider" is a good term to use here (LOB, external or customer are perspective dependent). "System" or already used in Sesam and we should not have the same name for two common concepts.

HubSpot is not a CRM system. HubSpot contains a CRM solution, but also other solutions such as a Marketing Hub and a Servieve Hub.

Could we add a sentence about why? I.e.

I think the term data type is ambiguous here. You refer to data type as in i.e. "person" or "company" but a data type generally refers to "bit", "boolean" or "array" or similar types. Should we find an other word here or does data type work?

What about: 
In this tutorial we will look closer at how to connect Sesam to two different data providers: `HubSpot <https://www.hubspot.com/>`_  and `Twelvedata <https://twelvedata.com/>`_. HubSpot is a SaaS provider and Twelvedata is a data provider og e.g. financial data such as stock information. After having succesfully connected to these providers inside Sesam you will create inbound pipes for each relevant datatype (potentially an other word here, but I don't not which) we are interested in to work on, as we go through a Sesam dataflow in the getting started guide.

Comment (JonasAls89) [reply tutorial-getting-started-collect.rst]: resolved. Djalil will push last changes.

File: tutorial-getting-started-collect.rst [hunk 2]
```diff
@@ -5,28 +5,127 @@
...
+
+After having successfully created both systems, you are now ready to move onto the next aspect of this tutorial, namely the creation of inbound pipes. 
+
+|
+|
+
+Create inbound pipes
+^^^^^^^^^^^^^^^^^^^^
+
+Inbound pipes are the naming convention used for pipes that receive their data from a source system.
```
Comment (erik-leven) [inline tutorial-getting-started-collect.rst:80]: "Inbound pipes" *is*

File: tutorial-getting-started-collect.rst [hunk 3]
```diff
@@ -5,28 +5,127 @@
...
+	  "_id": "crm",
+	  "type": "system:microservice",
+	  "docker": {
+	    "image": "sesamcommunity/learn-sesam-crm:v1.0.3",
+	    "memory": 512,
+	    "port": 5000
+	  }
+	}
+
+After having successfully created both systems, you are now ready to move onto the next aspect of this tutorial, namely the creation of inbound pipes.
```
Comment (erik-leven) [inline tutorial-getting-started-collect.rst:72]: Namely used above as well, maybe just ... tutorial: "Create inbound pipes"?

File: tutorial-getting-started-collect.rst [hunk 4]
```diff
@@ -5,28 +5,127 @@
...
+  
+  {
+    "_id": "twelvedata-stock-collect",
+    "type": "pipe",
+    "source": {
+      "type": "rest",
+      "system": "twelvedata",
+      "id_expression": "{{ exchange }}-{{ symbol }}",
+      "operation": "stocks",
+      "payload_property": "data",
```
Comment (erik-leven) [inline tutorial-getting-started-collect.rst:100]: comma needs to be removed

**Review by erik-leven — APPROVED** [review]

**Review by djalil-bouhachi — APPROVED** [review]

**Pull request merged**